config/sw-config.js

```diff
@@ -9,9 +9,8 @@ module.exports = {
       handler: "fastest",
       urlPattern: "\/$"
     }],
-    // context for script paths is the client folder
-    importScripts: ['./sw.js']
-    },
+    importScripts: ['./sw-events.js']
```

Comment:
`sw-registration` will do all the sw initialization, registration and handle states related to sw bootstrapping. 
`sw-events` will contain all the user defined event like push-notifications/background sync.